### PR TITLE
Added icelandic proficiency to frontend

### PR DESCRIPTION
**Pull request by 17luke**
@thdg 
Note, requires "icelandic_proficiency" column to be added to database clips table.
Note, please check my Icelandic in the localisation file (locales/isl/demographics.json)

- Asks user's Icelandic proficiency on demographic form.
- Automatically sets to "native" if mother tongue is Icelandic.
- uploadClip POST request now includes "icelandic_proficiency" in contribute-api.ts

### CODE REVIEW

**Comment by judyfong** [comment]
@17luke @thdg you should also check the proficiency level of their native tongue too. But more granualar: B2, C1, or C2. For example many native english and icelandic speakers are only at B2 or C1 level. They never bother getting to C2 level. 

Also you should change the why this matters text in english to match the icelandic one: the more varied and multi-faceted(?) multichanging?, the better. 

Finally when people are tired and have been doing an activity for over an hr their prociency at a task drops. 

Finally, you should check which region the speaker is from like rvk vs akureyri so you can evaluate recordings based off of their dialectal differencies not based off of absolute correctness, which is only akureyri/harðmæli dialect. 

A lot of kids recordings were rejected because they ellided some sounds even though its correct and native íslensku

File: public/static/locales/isl/demographics.json [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
     "gender": "Kyn",
     "native-tongue": "Móðurmál",
+    "icelandic-proficiency": "Íslenskukunnátta",
     "why-this-matters": "Hvers vegna skiptir þetta máli?",
     "why-this-matters-text": "Ofantaldar upplýsingar eru notaðar til að meta hversu lýðfræðilega dreift gagnasafnið Samrómur er. Því dreifðara og fjölbreyttara sem það er, því betra. Sjá skilmála og persónuverndaryfirlýsingu verkefnisins hér fyrir neðan til þess að fá frekari upplýsingar.",
     "accept-terms-conditions": "Ég staðfesti að hafa kynnt mér <2>skilmála</2> og <6>persónuverndaryfirlýsingu</5> verkefnisins",
     "consent-confirmed": "Leyfi staðfest",
     "switch-user": "Skipta um notanda",
+    "icelandic-proficiencies": {
+        "native": "Hefur móðurmálsþekkingu",
```
Comment (thdg) [inline public/static/locales/isl/demographics.json:12]: Should be "Hef móðurmálsþekkingu" for the user.
As in "Ég _hef móðurmálsþekkingu_", similar to "Ég er _byrjandi (A1-A2)_"

**Review by thdg — CHANGES_REQUESTED** [review]
This looks good.
@judyfong we decided on keeping it simple since people that do not know their A1-C2 level will not be good at judging the difference between for example A1 and A2 or C1 or C2. And also we want this to be quick and simple for the user.

File: public/static/locales/isl/demographics.json (hunk 1 above)
Comment (17luke) [reply public/static/locales/isl/demographics.json]: Changed to "Hef móðurmálsþekkingu" in d9e3f7e22a89f7c602897e1c9a08d73d26765528

**Pull request merged**